core/src/epicli/data/common/ansible/playbooks/roles/recovery/tasks/postgresql.yml

```diff
@@ -132,11 +129,8 @@
         service:
           name: "{{ repmgr_service_name[ansible_os_family] }}"
           state: started
-      when:
-        - component_vars.specification.extensions.replication.enabled is defined
-        - component_vars.specification.extensions.replication.enabled
-        - component_vars.specification.extensions.replication.use_repmgr is defined
-        - component_vars.specification.extensions.replication.use_repmgr
+        when:
```

Comment:
I just want to ensure that this `when` was moved intentionally.

Reply:
Fixed